content/en/blog/trust-in-your-wallet/index.md

```diff
@@ -0,0 +1,150 @@
+means proving one's identity in electronic services that meets specific requirements laid down
+by Finnish law. The identity providers are required to implement multi-factor authentication
+to identify the users. The result data of the authentication process typically contains
+the user's name and personal identification code. Often users authenticate using their bank ID.
+
+{{< imgproc identity Fit "1025x1025" >}}
+<em>Before the user starts the FTN authentication, he selects his bank in the identity service broker view.
+</em>
+{{< /imgproc >}}
+
+FTN is an exciting playground for us as it is so essential part of the digital processes
+of Finnish society. Integrating FTN in one way or another into the SSI wallet seems like a natural
+next step for us when thinking about how to utilize verified data large scale in Finland.
+
+Even though regulation may initially prohibit using SSI wallets as a
+“strong electronic identification” method, the FTN credential could be part of onboarding wallet
+users or provide additional security to services that currently use weak authentication.
+The selective disclosure feature would allow users to share only
+the needed data, e.g., age or name, without revealing sensitive personal identification code information.
+
+We decided to experiment with the FTN and create a PoC for a service (“FTN service”) that can issue
+credentials through an FTN authentication integration. And as it happened, we met our old friend
+OIDC protocol again.
+
+## The Demo
+
+The PoC FTN service can authenticate users through FTN identity broker integration and issue
+a credential for the received FTN data. The process starts with the user reading
+the FTN service connection invitation QR code with his SSI wallet.
+After the connection is ready,  the interaction with the FTN service and the user happens
+in the mobile device browser. FTN service instructs the user to authenticate with
+his bank credentials. After successful authentication, the FTN service receives
+the user information and issues a credential for the data.
+
+{{< youtube 7b4tKnSndxk >}}
+*The demo video shows how the technical PoC works.
+The user receives the FTN credential by authenticating himself with bank credentials.*
+
+Our previous OIDC demo provided data from the credentials through the OIDC protocol.
+In this FTN issuer demo, we utilized the OIDC protocol again but now used it as
+the credential data source. Once the user has acquired the FTN credential to his wallet,
+he can prove facts about himself without reauthenticating with his bank credentials.
+
+### The Sequence Flow
+
+The entire process sequence is below in detail:
+
+{{< figure src="/blog/2022/04/27/trust-in-your-wallet/sequence.svg" >}}
+*[Step-by-step sequence for the issuing process](https://github.com/findy-network/findy-issuer-tool/blob/master/api/README.md#ftn-credential-flow)*
+
+## Food for Thought: Self-service Issuers
```

Comment:
In the following three chapters, their style isn't as easy to read as it was at the beginning of the post. Maybe shorten the sentences or something. The points are excellent.